src/_styles/_noMachPage.scss

```diff
@@ -0,0 +1,23 @@
+.no-mach-page {
+  text-align: center;
+
+  h1 {
+    color: transparent;
```

Comment:
`color: transparent;` already stated in **h1-font-404** and will be added by this include `@include h1-font-404;` So no need to add it twice.